CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `InputAsset` and `OutputAsset` ([#361](https://github.com/Substra/substra/pull/361))
```

Comment:
I think it would be worth elaborating a bit ^^

Reply:
Which can of information about these do you think are relevant to mention?

Reply:
maybe just "added `inputAsset` blablaba" 
I didn't notice that it was part of the "added" section at the first glance, so I was a bit confused. 
Might be worth mentioning too that it's mirroring the backend schema 🤷